### PR TITLE
Add wholesome.js

### PR DESCRIPTION
**Pull request by TheOnlyGhostwolf**
### Description of Your Pull Request
Add new command - p!wholesome. This command grabs a random wholesome image from the internet

### Changes Proposed and or Files Changed
- Added wholesome.js
- Modified en-US.js

### Label Your Pull Request
- [ ] This PR modifies existing code to enhance the bot
- [ ] This PR fixes a bug within the bot
- [ ] This PR removes/renames methods or properties
- [x] This PR adds methods, features, commands or properties

### CODE REVIEW

**Comment by TheOnlyGhostwolf** [comment]
Mentioning @AdityaTD to remind him that this PR still exists

**Comment by MrJacz** [comment]
@TheOnlyGhostwolf does it work?

File: src/commands/Images/wholesome.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+const { Command, MessageEmbed } = require("../../index");
+const subReddits = ["wholesome", "aww", "AnimalsBeingBros"];
```
Comment (MrJacz) [inline src/commands/Images/wholesome.js]:
```suggestion

const SUB_REDDITS = ["wholesome", "aww", "AnimalsBeingBros"];
```

File: src/commands/Images/wholesome.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+module.exports = class extends Command {
+
+    constructor(...args) {
+        super(...args, {
+            cooldown: 5,
```
Comment (MrJacz) [inline src/commands/Images/wholesome.js]:
```suggestion
            cooldown: 10,
            bucket: 2,
```

File: src/commands/Images/wholesome.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+            extendedHelp: "No extended help available."
+        });
+    }
+
+    async run(msg) {
+        const subReddit = subReddits[Math.floor(Math.random() * subReddits.length)];
+        let img = await this.client.funcs.scrapeSubreddit(subReddit);
+        if (img && img.includes(".mp4")) img = await this.client.funcs.scrapeSubreddit(subReddit);
+        if (!img || img.includes(".mp4")) return msg.sendMessage(`Too fast, too furious, try again!`);
+        return msg.sendEmbed(new MessageEmbed().setImage(img).setFooter(`/r/${subReddit} - PenguBot.com`));
```
Comment (MrJacz) [inline src/commands/Images/wholesome.js]:
```suggestion
        return msg.sendEmbed(new MessageEmbed()
            .setImage(img)
            .setFooter(`/r/${subReddit} - PenguBot.com`)
        );
```

File: src/languages/en-US.js [hunk 4]
```diff
@@ -460,6 +460,7 @@ module.exports = class extends Language {
             COMMAND_MODLOG_DESCRIPTION: "Change the Mod Logs setting in the server.",
 
             COMMAND_MEME_DESCRIPTION: "Grabs you a random meme from the interwebs.",
+            COMMAND_WHOLESOME_DESCRIPTION: "Grabs you a random wholesome image from the interwebs.",
```
Comment (MrJacz) [inline src/languages/en-US.js]: Add this key to all other languages, just keep it in english.

**Comment by TheOnlyGhostwolf** [comment]
@MrJacz Fulfilled the changes you've requested

**Pull request merged**